roomClass.py

```diff
@@ -1,33 +1,25 @@
-class room:
+from enum import Enum
+
+class RoomStatus(Enum):
+    UNAVAILABLE=0
+    VACANT = 1
+    CLEANING_NEEDED=2
+
+class Room:
     def __init__(self, number, info, price, status, time=None):
-        self.number = number
-        self.info = info
+        self.room_number = number
+        self.info = info # What info? Your variables need to be more descriptive
         self.price = price
         self.status = status
-        if(time!=None):
-            #*use datetime to implement date subtraction to check if reservation has expired
-            self.time = time
-        else:
-            self.time = None
-    def changeStatus(self,status):
-        self.status = status
-    def changeTime(self,time):
-        self.time = time
-    def checkNum(self):
-        return self.number
-    def checkStatus(self):
-        return self.status
-    def checkTime(self):
-        return self.time
+        self.checkout_time = time
 
-    def printInfo(self, printAll=False):
-        if(printAll == True):
-            if(self.status!='VACANT'):
-                print("UNAVAILABLE",end=" ")
-            print(self.number, '-', self.info, "$"+self.price+'/night',self.status, self.time, sep=" ")
+    def __str__(self):
+        if self.status == RoomStatus.VACANT:
+            return (f'{self.room_number} - {self.info} ${self.price}/night')
+        elif self.status == RoomStatus.UNAVAILABLE:
+            return ("Unavailable")
         else:
-            if((self.status=='VACANT')):
-                print(self.number, '-', self.info, "$"+self.price+'/night', sep=" ")
+            return(f'{self.room_number} - {self.info} ${self.price}/night {self.status} {self.checkout_time}')

```

Comment:
What you're doing with printInfo is already supported in Python. The `__str__` function is allows you to implement your own way of converting a Python object into string format. Its cleaner because to print the object you don't have to do `print(room.printInfo())` you can just do `print(room)`. The presence of the `__` is a hint that these functions are intended to be invoked indirectly.